data/json/items/armor.json

```diff
@@ -10594,26 +10594,34 @@
     {
         "type" : "ARMOR",
         "id" : "legrig",
-        "name" : "pair of drop leg pouches",
-        "name_plural" : "pairs of drop leg pouches",
-        "weight" : 205,
+        "name" : "drop leg ammo pouch",
+        "name_plural" : "drop leg ammo pouches",
+        "weight" : 100,
         "color" : "dark_gray",
-        "covers" : ["LEGS"],
+        "covers" : ["LEG_EITHER"],
         "to_hit" : 0,
-        "storage" : 8,
+        "storage" : 0,
         "symbol" : "[",
-        "description" : "A set of pouches that can be worn on the thighs using buckled straps.  This variety is favored by the military.",
-        "price" : 3000,
+        "description" : "An ammo pouch that can be worn on the thigh using buckled straps.  This variety is favored by the military.",
+        "price" : 2000,
         "material" : ["cotton", "null"],
-        "volume" : 2,
+        "volume" : 1,
         "cutting" : 0,
         "warmth" : 0,
         "phase" : "solid",
-        "encumbrance" : 5,
+        "encumbrance" : 2,
         "bashing" : -3,
```

Comment:
It is ignored anyway, because it doesn't make much sense. It only ever makes sense to attack shockers with items with stats this bad, for the rest fists are better.
In `item::damage_bash/cut`, only non-negative values are returned.